0.6.7/index.html

```diff
@@ -19,6 +20,46 @@
     </script>
     <script type="text/javascript" src="_static/jquery.js"></script>
     <script type="text/javascript" src="_static/doctools.js"></script>
+    <script type="text/javascript">
+    $(function () {
+        $.ajax({
+           url: 'http://docs.sympy.org/releases.txt',
+           success: function(data){
+            var lines = data.split("\n");
+            $.each(lines, function(n, elem) {
+                if (elem != ""){
+                var res = elem.split(":");
+                  if(res[0] == "0.7.6.1"){
+                    $("#version-list").prepend("<li><a style='font-size:100%;' href="+"../latest/index.html"+"><span>Latest</span></a></li>");
+                  }
```

Comment:
That file is generated automatically from the release script. Would it be possible to just check what the last value in the file is, and set that to "latest"?